### PR TITLE
Multi key

### PR DESCRIPTION
**Pull request by kenarsa**

### CODE REVIEW

File: binding/python/porcupine.py [hunk 1]
```diff
@@ -39,15 +39,28 @@ class PicovoiceStatuses(Enum):
     class CPorcupine(Structure):
         pass
 
-    def __init__(self, library_path, model_file_path, keyword_file_path, sensitivity):
+    def __init__(
+            self,
+            library_path,
+            model_file_path,
```
Comment (amiranhari) [inline binding/python/porcupine.py]: you can keep the interface the same and in the body of your function have
`if isinstance(model_file_path, list):
     model_file_paths = model_file_path
  else:
    model_file_paths = list(model_file_path)
`

Same for sensitivity.

Comment (kenarsa) [reply binding/python/porcupine.py]: that's an option. I prefer it this way as the argument names go with semantics.

File: demo/python/porcupine_demo.py [hunk 2]
```diff
@@ -185,13 +192,13 @@ def _default_library_path():
...
-        if not args.keyword_file_path:
-            raise ValueError('keyword file path is missing')
+        if not args.keyword_file_paths:
+            raise ValueError('keyword file paths are missing')
 
         PorcupineDemo(
             library_path=args.library_path,
             model_file_path=args.model_file_path,
-            keyword_file_path=args.keyword_file_path,
+            keyword_file_paths=args.keyword_file_paths.split(','),
```
Comment (amiranhari) [inline demo/python/porcupine_demo.py]: do this so to also strip the whitespaces: `[x.strip() for x in my_string.split(',')]`

File: binding/android/.idea/misc.xml [hunk 3]
```diff
@@ -5,11 +5,12 @@
     <option name="myDefaultNotNull" value="android.support.annotation.NonNull" />
```
Comment (amiranhari) [inline binding/android/.idea/misc.xml]: I don't think we should push this file to repo. Are we ignoring the whole .idea directory in .gitignore?

File: demo/python/porcupine_demo.py (hunk 2 above)
Comment (kenarsa) [reply demo/python/porcupine_demo.py]: good point. addressed.

File: binding/android/.idea/misc.xml (hunk 3 above)
Comment (kenarsa) [reply binding/android/.idea/misc.xml]: No. only these two sub-folders:

/.idea/workspace.xml
/.idea/libraries

why?

File: binding/ios/PorcupineManager.swift [hunk 4]
```diff
@@ -74,9 +81,26 @@ class PorcupineManager {
     ///   - keywordCallback: Callback to be executed after wake word detection.
     init(modelFilePath: String, keywordFilePath: String, sensitivity: Float, keywordCallback: @escaping (() -> Void)) {
```
Comment (amiranhari) [inline binding/ios/PorcupineManager.swift]: Please check the parameters. I think we have duplicate modelFilePath

File: binding/ios/PorcupineManager.swift [hunk 5]
```diff
@@ -74,9 +81,26 @@ class PorcupineManager {
...
+
+    
+    /// Initializer for multiple keyword use case.
+    ///
+    /// - Parameters:
+    ///   - modelFilePath: Absolute path to file containing model parameters.
+    ///   - keywordFilePaths: Absolute paths to keyword files.
+    ///   - sensitivities: List of sensitivity parameters for each keyword.
+    ///   - keywordCallback: Callback to be executed after wake word detection.
+    init(modelFilePath: String, keywordFilePaths: [String], sensitivities: [Float], keywordCallback: @escaping ((Int32) -> Void)) {
```
Comment (amiranhari) [inline binding/ios/PorcupineManager.swift]: Also here we don't need modelFilePath only modelFilePaths.

**Pull request merged**